File: app/src/main/java/com/crossbowffs/quotelock/history/app/QuoteHistoryActivity.java
package com.crossbowffs.quotelock.history.app;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.crossbowffs.quotelock.R;

/**
 * @author Yubyf
 */
public class QuoteHistoryActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_main);

        getSupportFragmentManager()
                .beginTransaction()
                .replace(R.id.content_frame, new QuoteHistoryFragment())
                .commit();
    }
}

File: app/src/main/java/com/crossbowffs/quotelock/app/history/QuoteHistoryActivity.kt
package com.crossbowffs.quotelock.app.history

import android.os.Bundle
import androidx.activity.compose.setContent
import androidx.appcompat.app.AppCompatActivity
import com.crossbowffs.quotelock.ui.navigation.QuoteNavHost
import com.crossbowffs.quotelock.ui.theme.QuoteLockTheme
import dagger.hilt.android.AndroidEntryPoint

/**
 * @author Yubyf
 */
@AndroidEntryPoint
class QuoteHistoryActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContent {
            QuoteLockTheme {
                QuoteNavHost {
                    finish()
                }
            }
        }
    }
}